lib/matplotlib/tests/test_patches.py

```diff
@@ -410,3 +410,23 @@ def test_contains_points():
     expected = path.contains_points(points, transform, radius)
     result = ell.contains_points(points)
     assert np.all(result == expected)
+
+    
+def test_when_update_from_and_set_alpha_then_only_alpha_changes():
+    # given
+    source_facecolor = (0.234, 0.123, 0.135, 0.322)
+    source_egdecolor = (0.728, 0.682, 0.945, 0.268)
+    source = mpatches.Rectangle((0, 0), 1., 1., facecolor=source_facecolor, edgecolor=source_egdecolor)
```

Comment:
Could you make this line and the one below 79 characters or less long by putting some of the arguments on the next line (and lining them up with the brackets?). Otherwise this looks good to go 👍